### ISSUE TITLE
Mask Loss

### ISSUE THREAD

**Issue by Xuanphu108** [akwasigroch/coattention_object_segmentation#1]
I think that mask loss is also important in this work. Can you implement the mask loss?

**Comment by akwasigroch** [comment]
Yes, the mask loss is important, because it increases the performance of the system. However, I no longer develop this algorithm, due to lack of time
